test/integration-serial/serial_deploy_test.rb

```diff
@@ -603,6 +603,24 @@ def test_global_deploy_validation_catches_namespaced_cr
     wait_for_all_crd_deletion
   end
 
+  def test_global_deploy_prune_black_box_success
+    setup_template_dir("globals") do |target_dir|
+      flags = "-f #{target_dir} --selector app=krane"
+      out, err, status = krane_black_box("global-deploy", "#{KubeclientHelper::TEST_CONTEXT} #{flags}")
+      assert_empty(out)
+      assert_match("Success", err)
+      assert_predicate(status, :success?)
+
```

Comment:
flush the logger

Reply:
krane_black_box uses `Open3.capture3` so each call get's a unique stream